### DISCUSSION TITLE
2023-05-24 Weekly Hosting Meeting

### DISCUSSION THREAD

**Discussion by Crixu** [WordPress/hosting-handbook#148] [Weekly Meetings]
# Meeting

/here < meeting type="hosting" > Who’s here to chat about hosting?

👋 Hello! This is the weekly meeting for the WordPress.org Hosting Team, who works with hosts and developers to help improve the WordPress user experience. This is a great time to ask any questions you may have, or visit Make/Hosting for more info. 

🥇 If you participate in the Five for the Future program, drop your WordPress.org username in a thread here 👉

🤗 If you're new here, welcome! You are welcome to introduce yourself if you like in a thread here 👉

-----

## Agenda

📓 ··· Here is the agenda for this meeting ···

**WordPress Community**
- WordPress 6.2.2
- 20 Days to Celebrate 20 Years of WordPress, From Blogs to Blocks
- WordPress Contributor Mentorship Program

**Hosting Team Time**
- wp-now streamlines the process of setting up a local WordPress environment
- Contributor Days

**Hosting Team Projects**
- Advanced Administration Handbook
- Host Test Runners

**Open Floor**
- Anything not in the agenda

Would anyone like to add to or change any items on the agenda?

OK, let’s get started with the WordPress Community news.

-----

### :wordpress: ··· WordPress 6.2.2 ···
[WordPress 6.2.2 Security Release – WordPress News](https://wordpress.org/news/2023/05/wordpress-6-2-2-security-release/)
After the release of WordPress 6.2.1 last week we also got another bugfix release at the end of last week. The release should fix the issue with shortcodes not rendering inside a block theme. 

Anyone had issues rolling out the second release last week? 

### :community: ··· 20 Days to Celebrate 20 Years of WordPress, From Blogs to Blocks ···

[20 Days to Celebrate 20 Years of WordPress, From Blogs to Blocks](https://make.wordpress.org/marketing/2023/05/02/20-days-to-celebrate-20-years-of-wordpress-from-blogs-to-blocks/)

We are so happy to be celebrating WordPress and the incredible WordPress community as we approach the 20th anniversary of WordPress on May 27, 2023. We hope you enjoy participating in this community game and that it gives you an opportunity to reflect on and share your favorite parts of WordPress. 

Who already joined a birthday party or is planning to join one? 

### :community: ··· WordPress Contributor Mentorship Program ···

[WordPress Contributor Mentorship Program: Pilot Program Proposal – WordPress.org Project](https://make.wordpress.org/project/2023/05/17/wordpress-contributor-mentorship-program-pilot-program-proposal/#comment-286)

In February 2023, a project-wide WordPress mentorship program was proposed and a working group was established to create this program. The program provides mentorship in a cohort-based format or a one-on-one setting, offering guidance, skills, and knowledge to help new contributors. The program aims to assist contributors in making their first contributions, select their area of participation, build community connections, and gain success in their careers through open source contributions.

The contributor working group is currently seeking feedback on the program and invites interested members of the community to join as mentors, provide feedback on the program, or assist in its development.

-----

### :computer: ···  wp-now ···

[@wp-now](https://www.npmjs.com/package/@wp-now/wp-now)

wp-now streamlines the process of setting up a local WordPress environment.

It uses automatic mode detection to provide a fast setup process, regardless of whether you're working on a plugin or an entire site. You can easily switch between PHP and WordPress versions with just a configuration argument. Under the hood, wp-now is powered by WordPress Playground and only requires Node.js.

### 🗓️ ···  Hosting Contributor Day ···

Heads up, some events are coming down the pipeline along with some Contributor Days to participate in:

- 2023-06-08, [WordCamp Europe](https://europe.wordcamp.org/2023/), Athens (Greece).
- 2023-08-24, [WordCamp US](https://us.wordcamp.org/2023/), National Harbor, Maryland (USA).

Is anyone [going to a WordCamp](https://central.wordcamp.org/schedule/) and wants to lead a Hosting Table?

-----

Great, so just a look at our projects.

### 📕 ··· Advanced Administration Handbook ···

- [Public Handbook](https://developer.wordpress.org/advanced-administration/)
- [GitHub Handbook](https://github.com/WordPress/Advanced-administration-handbook)

Currently, [everything pending to do is in the Issues part](https://github.com/WordPress/Advanced-administration-handbook/issues). A lot of ["documentation + enhancement"](https://github.com/WordPress/Advanced-administration-handbook/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) tickets, and also some ["documentation + good first issue"](https://github.com/WordPress/Advanced-administration-handbook/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) for Contributor Day.

If anybody needs help or mentoring for this project, @javiercasares is always available.

### :cloud: ··· Host Test Runners ···

The idea behind the [distributed hosting tests](https://make.wordpress.org/hosting/test-results/) is to run WordPress' automated test suite across a variety of hosting environments, to help verify that changes on WordPress don't unintentionally affect hosts, and to help hosts be aware of how changes on WordPress or their environments affect how WordPress runs on their services.

Hosts that report receive automated emails when the tests are passing on WordPress.org, but failing on their environment so that they can investigate.

We want to make sure that as many hosts as possible are included in the Host Test Results database, so that WordPress users, the #core-test, and the #core team can benefit from this valuable resource. That's why we're asking you to check if your host is already supporting Host Test Results, and to add your host if it still needs to be included.

To check if your hoster is already supporting Host Test Results, visit the website (https://make.wordpress.org/hosting/test-results/) and search for your host's name. If your host is already included, great! You can use the information provided to make sure that you're getting the best possible performance and reliability from your site. 

If your host isn't included, don't worry, you can still help! You can just follow the instructions on the website to add your host to the database. It's quick and easy, and you'll be helping other WordPress users find the best hosts for their sites. However, if your host has not received reports in the last 25 revisions, we encourage you to check back with them.

- [Try the PHPUnit Test Runner](https://make.wordpress.org/hosting/handbook/tests/#try-the-phpunit-test-runner)
- [How to report: Creating your bot for WordPress.org](https://make.wordpress.org/hosting/handbook/tests/#how-to-report-creating-your-bot-for-wordpress-org)

-----

## Open Floor

Ok, if nothing else, let’s move on to 🤖 Open Floor. Folks are welcome to bring any topic up for discussion at this time.

-----

## Next meeting

⏰ Here are the [team's upcoming meetings](https://make.wordpress.org/meetings/#hosting):

- 2023-05-24 at 18:00 UTC (in a few hours)
- 2023-05-31 at 09:00 UTC (next week)
- 2023-05-31 at 18:00 UTC

Note: Due to travels to WordCamp Europe we will skip the early meeting on 2023-06-07 at 09:00 UTC (in two weeks) 

📌 Okay, we're out of time for today, let's close out this meeting.

You are welcome to continue the conversation here in the #hosting-community channel or on Make/Hosting, at any time.

Thank you all for joining and for your contributions to WordPress!
See you at the next Hosting Team weekly meeting! < /meeting >

**Comment by jessfrick** [comment]
Five for the future peeps: 

- jasonways
- baxbridge
- jessibelle

Thanks for doing the notes @Crixu !
